### PR TITLE
Add ability to get current non cli- tag

### PR DESCRIPTION
**Pull request by andriisoldatenko**
+ fixed 2 warnings.
fix astronomer/issues#477
related PR https://github.com/astronomer/astro-cli/pull/303

### CODE REVIEW

File: src/resolvers/types/deployment/index.js [hunk 1]
```diff
@@ -97,7 +97,17 @@ export async function deployInfo(parent, args, ctx) {
...
+  const imagesCreated = await ctx.db.query.dockerImages(
+    {
+      where: { deployment: { id: parent.id } },
+      order: [["created", "DESC"]],
+      limit: 1
+    },
+    `{ tag }`
+  );
+  let currentTag = map(imagesCreated, "tag")[0];
+  return { latest, next, currentTag };
```
Comment (schnie) [inline src/resolvers/types/deployment/index.js]: @andriisoldatenko feels like we may want to figure out a better way to name these with the new addition. 

Maybe we stay consistent and omit `tag`, or find better names altogether. We'd need to update clients though.

File: src/resolvers/types/deployment/index.js [hunk 2]
```diff
@@ -97,7 +97,17 @@ export async function deployInfo(parent, args, ctx) {
...
+
+  const imagesCreated = await ctx.db.query.dockerImages(
+    {
+      where: { deployment: { id: parent.id } },
+      order: [["created", "DESC"]],
+      limit: 1
+    },
+    `{ tag }`
+  );
+  let currentTag = map(imagesCreated, "tag")[0];
```
Comment (schnie) [inline src/resolvers/types/deployment/index.js]:
```suggestion
  const current = map(imagesCreated, "tag")[0];
```

File: src/resolvers/types/deployment/index.js (hunk 1 above)
Comment (schnie) [inline src/resolvers/types/deployment/index.js]:
```suggestion
  return { latest, next, current };
```

File: src/schema.graphql [hunk 3]
```diff
@@ -167,6 +167,7 @@ type DeploymentUrl {
 type DeployInfo {
   latest: String
   next: String
+  currentTag: String
```
Comment (schnie) [inline src/schema.graphql]:
```suggestion
  current: String
```

Comment (ashb) [reply src/schema.graphql]: We could use `@deprecated` and change this to be

```graphql
type DeployInfo {
  current: String
  nextCli: String
  latest: String @deprecated
  next: String @deprecate @alias(field: "nextCli")
```

(I feel we don't need `latest` which is the latest `cli-` tagged one anymore

Comment (andriisoldatenko) [reply src/schema.graphql]: @ashb @schnie fixed!

File: src/resolvers/types/deployment/index.js (hunk 1 above)
Comment (andriisoldatenko) [reply src/resolvers/types/deployment/index.js]: fixed.

File: src/resolvers/types/deployment/index.js (hunk 2 above)
Comment (andriisoldatenko) [reply src/resolvers/types/deployment/index.js]: fixed.

**Comment by andriisoldatenko** [comment]
@ashb @schnie PTAL

File: src/resolvers/types/deployment/index.js [hunk 4]
```diff
@@ -97,7 +97,17 @@ export async function deployInfo(parent, args, ctx) {
...
+  const imagesCreated = await ctx.db.query.dockerImages(
+    {
+      where: { deployment: { id: parent.id } },
+      order: [["created", "DESC"]],
+      limit: 1
+    },
+    `{ tag }`
+  );
+  const current = map(imagesCreated, "tag")[0];
+  return { latest, next, current };
```
Comment (schnie) [inline src/resolvers/types/deployment/index.js]: @andriisoldatenko don't we need to update here too?

File: src/schema.graphql [hunk 5]
```diff
@@ -165,8 +165,10 @@ type DeploymentUrl {
 }
 
 type DeployInfo {
-  latest: String
-  next: String
+  current: String
+  nextCli: String
+  latest: String @deprecated
+  next: String @deprecated @alias(field: "nextCli")
```
Comment (schnie) [inline src/schema.graphql]: @samblackk do we use these fields in orbit? Anything we need to update there?

File: src/resolvers/types/deployment/index.js (hunk 4 above)
Comment (andriisoldatenko) [reply src/resolvers/types/deployment/index.js]: fixed.

Comment (andriisoldatenko) [reply src/resolvers/types/deployment/index.js]: fixed.

**Pull request merged**